components/search_results.jsx

```diff
@@ -383,7 +383,24 @@ export default class SearchResults extends React.Component {
                 </div>
             );
         } else {
-            ctls = results.order.map(function searchResults(id, idx, arr) {
+            const order = [...results.order];
```

Comment:
For efficiency we could skip the array copy if it's not going to be used (if `!this.props.isPinnedPosts`). 

Just wondering if we can use the short form sort like below if `create_at` is a timestamp... haven't tried it but just a thought:
```javascript
(a, b) => results.posts[a].create_at - results.posts[b].create_at
```

Reply:
Good call, updates made